master/reference/calicoctl/resources/policy.md

```diff
@@ -48,7 +48,7 @@ spec:
 
 | Field    | Description                 | Accepted Values   | Schema | Default    |
 |----------|-----------------------------|-------------------|--------|------------|
-| order    | Indicates priority of this policy, with lower order taking precedence. | | integer | 100 |
+| order    | Indicates priority of this policy, with lower order taking precedence.  No value indicates highest order (lowest precedence) | | float | unassigned (highest order) |
```

Comment:
Not sure if I understand the Default value field `unassigned (highest order)` does it mean it's unassigned by default? or `unassigned highest order`